styles/_mandara-insight.scss

```
@@ -0,0 +1,1461 @@
+  font-weight: bold;
+  font-size: 14px;
+}
+
+// Local markdown preview
+.local-markdown {
+  background: rgba(0, 0, 0, 0.4);
+  border: 1px solid rgba(255, 255, 255, 0.08);
+  border-radius: 6px;
+  padding: 14px;
+  color: rgba(255, 255, 255, 0.8);
+  font-family: "SFMono-Regular", Consolas, monospace;
```

Stylelint (17.6.0)

[error] Unexpected quotes around "SFMono-Regular" (font-family-name-quotes)

(font-family-name-quotes)